src/targets/adminCreateUser.ts

```diff
@@ -105,7 +105,13 @@ export const AdminCreateUser =
   async (ctx, req) => {
     const userPool = await cognito.getUserPool(ctx, req.UserPoolId);
     const existingUser = await userPool.getUserByUsername(ctx, req.Username);
-    if (existingUser && req.MessageAction === "RESEND") {
+    const supressWelcomeMessage =
+      req.MessageAction === "SUPRESS";
```

Comment:
SUPRESS is typoed. Need to be spelt "SUPPRESS"

Reply:
thx